rtv/subreddit_page.py

```diff
@@ -74,7 +74,10 @@ def prompt_subreddit(self):
 
         name = self.term.prompt_input('Enter page: /')
         if name is not None:
-            self.refresh_content(order='ignore', name=name)
+            if not '/comments/' in name:
+                self.refresh_content(order='ignore', name=name)
+            else:
+                self.open_submission(url='https://www.reddit.com/r/'+name)
```

Comment:
Hadn't thought of that. I'll add some better parsing/look to see if PRAW has something built in.